desktop/components/commercial_filter/models/params.coffee

```diff
@@ -57,7 +60,14 @@ module.exports = class Params extends Backbone.Model
     _.uniq(@fullyQualifiedLocations.concat((@get('aggregation_partner_cities') || [])).concat @get('partner_cities'))
 
   whitelisted: ->
-    whitelisted = _.pick @current(), @urlWhitelist
-    omitted = _.omit whitelisted, (val, key) ->
+    paramsFromUrl = qs.parse(location.search.replace(/^\?/, ''))
+    whitelistedUrlParams = _.pick paramsFromUrl, (value, key) ->
```

Comment:
the `value, key` arg ordering is quite counter-intuitive to me... strange interface.